frontend/src/components/experimentRuns/ExperimentRunsProperties.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import {connect} from "react-redux";
+
+import {Table, Typography} from "antd";
+const {Title} = Typography;
+
+const mapStateToProps = state => ({
+  propertyValuesByID: state.propertyValues.itemsByID,
+});
+
+const actionCreators = {};
+
+const ExperimentRunsProperties = ({
+  propertyValuesByID,
+  propertyIDs,
+  protocolName,
+}) => {
+  let properties = []
+  let columns = []
+  let data = []
+
+  if ( propertyValuesByID[propertyIDs[0]]) {
```

Comment:
space after parenthesis and is this a test to check if the protocol has properties ? i would create a intermediate variable hasProperties or hasPropertyValues for readability.